facts/config.py

```diff
@@ -45,22 +45,35 @@
     'clotoVersion': 'v1.0',                 # Cloto API version
     'redisHost':    'localhost',            # host of Redis
     'redisQueue':   'policymanager',        # name of the queue in redis
-    'rabbitMQ':     'localhost',        # IP of the RabbitMQ server
-    'cloto':        'localhost',             # IP of fiware-cloto component
+    'rabbitMQ':     'localhost',            # IP of the RabbitMQ server
+    'cloto':        '127.0.0.1',            # IP of fiware-cloto component
```

Comment:
Because, cloto component accepts requests from 127.0.0.1 but does not accept localhost as default